proposal/Cycatz-nycu/README.org

```diff
@@ -0,0 +1,83 @@
+* Uniramp 
+** Basic Information
+Uniramp is a tool for analyzing the gray values of glyphs in a given font; it also provides the image-to-Unicode conversion as a showcase.
+
+** Problem to solve
+Ascii-art has been in its place since the 1970s. Nowadays, we still can see this kind of art on BBS (bulletin board systems), emails, etc.
+As Unicode Standard is defined to include more and more characters, it's more and more flexible for those designers to choose suitable characters to draw. (In fact, this is not ascii-art anymore, maybe it should be called "Unicode-art"?)
+
+Additionally, some methods simplifying the process of manually translating an image from pixels to bytes. One of them is the "Image to text conversion" technique. This method is to sample the image down to grayscale with less than 8-bit precision, and then assign a character for each value. For example, the most commonly-used "character ramp" for asciis is the sequence ="$@B%8&WM#*oahkbdpqwmZO0QLCJUYXzcvunxrjft/\|()1{}[]?-_+~<>i!lI;:,"^`'. "=, from black to white. 
+
+However, most of the "Image to text" conversion tools online do not support custom character sets and Unicode codepoints. They usually implement it with the standard character ramp as mentioned above since it does not require additional character and font information. It's so bad that it does not fully utilize over 100000 code points defined in Unicode.   
+
+Therefore, this project aims to generate a Unicode-version art with the given image and font file. And the user can specify their own character set, which might contain CJK or emojis characters, to indicate the generated text art must only contain these characters. Furthermore, this program also displays the character ramp with each glyph's "coverage", which is defined as ~coverage = area of the shape / (glyph advance width * units per em)~, for doing further analysis.
```

Comment:
AA is very useful for code remarks.  I think an optional ASCII mode makes a lot of sense when doing the unicode drawing.